src/lib/uaa/uaa.ts

```diff
@@ -192,7 +192,7 @@ export default class UAAClient {
     try {
       const response = await this.request('get', `/Users/${userGUID}`);
       return response.data as types.IUaaUser;
-    } catch (err) {
+    } catch (err: any) {
```

Comment:
Instead of `any`, do you think this could be `AxiosError`?

https://github.com/axios/axios/blob/master/index.d.ts#L92-L99

Reply:
@paroxp that would make sense, should have checked Axios'types
though, it's not happy 
![Screenshot 2021-08-31 at 12 23 01](https://user-images.githubusercontent.com/3758555/131494534-f1ad1efd-f125-4613-bfa5-998b1b75f7c4.png)

Reply:
ah more info at https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#use-unknown-catch-variables

Reply:
Weird. As you were! Thanks for checking this out.

Reply:
Looking at example they have
 
```
if (err instanceof Error) {
     console.error(err.message);
 }
```
which wouldn't exactly work in our case, but i we can somehow defien a custom type that extends `AxiosError` ?